microsoft-edge/developer/index.yml

```diff
@@ -32,70 +32,165 @@ metadata:
+
+          - text: Microsoft Edge DevTools documentation  # DevTools landing page
+            url: /microsoft-edge/devtools-guide-chromium/landing/index
+
+          - text: Overview of DevTools  # renamed from index.md
+            url: /microsoft-edge/devtools-guide-chromium/overview
+
           - text: DevTools for beginners
             url: /microsoft-edge/devtools-guide-chromium/beginners/html
+
+      - linkListType: whats-new
+        links:
+          - text: What's new in Microsoft Edge DevTools
+            url: /microsoft-edge/devtools-guide-chromium/whats-new/whats-new
+
       - linkListType: how-to-guide
         links:
           - text: Command Menu
             url: /microsoft-edge/devtools-guide-chromium/command-menu/index
-          - text: Open DevTools
-            url: /microsoft-edge/devtools-guide-chromium/open
+
           - text: View and change CSS
             url: /microsoft-edge/devtools-guide-chromium/css/index
+
       - linkListType: reference
         links:
           - text: Keyboard shortcuts
             url: /microsoft-edge/devtools-guide-chromium/shortcuts
-  # Card
+
+# =============================================================================
+# Card
   - title: Microsoft Edge extensions
     linkLists:
       - linkListType: overview
         links:
           - text: Extensions overview
             url: /microsoft-edge/extensions-chromium/index
+
           - text: Getting started tutorial
             url: /microsoft-edge/extensions-chromium/getting-started/index
+
           - text: Port a Chromium extension to Microsoft Edge
             url: /microsoft-edge/extensions-chromium/developer-guide/port-chrome-extension
       - linkListType: how-to-guide
         links:
           - text: Alternate methods of distributing extensions
             url: /microsoft-edge/extensions-chromium/developer-guide/alternate-distribution-options
+
           - text: Step-wise publishing process
             url: /microsoft-edge/extensions-chromium/publish/publish-extension
+
           - text: Register as a Microsoft Edge extension developer
             url: /microsoft-edge/extensions-chromium/publish/create-dev-account
+
           - text: Create a simple extension
             url: /microsoft-edge/extensions-chromium/getting-started/part1-simple-extension
-  # Card
+
+
+# =============================================================================
+# Card
   - title: Progressive Web Apps
     linkLists:
       - linkListType: overview
         links:
           - text: Progressive Web Apps overview
             url: /microsoft-edge/progressive-web-apps-chromium/index
+
           - text: Get started with Progressive Web Apps
             url: /microsoft-edge/progressive-web-apps-chromium/how-to/index
+
           - text: Debug Progressive Web Apps
             url: /microsoft-edge/devtools-guide-chromium/progressive-web-apps
+
           - text: What's New in Progressive Web Apps
```

Comment:
What do you think about moving this inside a `linkListType: whats-new` card section? For better visibility.